sycl/include/sycl/ext/oneapi/kernel_properties/properties.hpp

```diff
@@ -138,6 +159,48 @@ template <> struct property_value<single_task_kernel_key> {
   using key_t = single_task_kernel_key;
 };
 
+template <size_t Dim0, size_t... Dims>
+struct property_value<max_work_group_size_key,
+                      std::integral_constant<size_t, Dim0>,
+                      std::integral_constant<size_t, Dims>...> {
```

Comment:
If we indeed want this to always take 3 values (we can relax it later) then this should be made to take exactly 3 template arguments of type `size_t`.

Reply:
Yep, good idea - thanks

Reply:
Done, cheers